test/Templates.Test/EmptyWebTemplateTest.cs

```diff
@@ -2,9 +2,11 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Testing.xunit;
+using Templates.Test.Helpers;
 using Xunit;
 using Xunit.Abstractions;
 
+[assembly: AssemblyFixture(typeof(SeleniumServerFixture))]
```

Comment:
This looks cool. Didn't know this existed. I think we'll copy this technique in the Blazor repo too!

Reply:
It may actually be possible to have it in our Common repo and use the same code!